app/src/main/resources/db/migration/V1__create_provider_item_sub_item_tables.sql

```diff
@@ -0,0 +1,48 @@
+CREATE TABLE IF NOT EXISTS "provider-service".provider
+(
+    id bigint NOT NULL GENERATED ALWAYS AS IDENTITY ( INCREMENT 1 START 1000 MINVALUE 1 MAXVALUE 9223372036854775807 CACHE 1 ),
```

Comment:
Please put create a sequence as a separate query, this way you can easily manage the name of it.
`MAXVALUE` is not needed